ReactNativeImageCropping.m

```diff
@@ -88,8 +88,24 @@ - (NSDictionary *)constantsToExport
 
 - (void)handleImageLoad:(UIImage *)image {
     
+    // hardcoded downsample of image
```

Comment:
Hi sorry, I did not realize further commits would modify this pull request. I think it would be helpful to let users set a max resolution though because loading a cropped full resolution photo from the camera album has a noticeable delay on my 6S. I can add that feature or just close this pull request.